app/controllers/application_controller.rb

```diff
@@ -1,7 +1,8 @@
 class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
-  protect_from_forgery with: :exception
+  #protect_from_forgery with: :exception
```

Comment:
I think we probably want to keep this. Was this commented out for debugging, or is there another issue going on?